cypress/support/fragments/settings/circulation/fee-fine/overdueFinePolicies.js

```diff
@@ -0,0 +1,179 @@
+const intervalSelect = Select({ name: 'overdueFine.intervalId' });
+const generalFeePolicySection = Section({ id: 'generalFeePolicy' });
+const feeViewSection = Section({ id: 'viewFineSection' });
+
+export default {
+  waitLoading() {
+    cy.expect([
+      Link(mainTitle).exists(),
+      Heading(mainTitle).exists(),
+    ]);
+  },
+
+  openCreatingForm() {
+    cy.do([
+      actionsButton.click()
+    ]);
+  },
+
+  checkCreatingForm() {
+    cy.do([
+      createFormSection.click()
+    ]);
+    cy.expect([
+      createFormSection.exists(),
+      createFormHeader.exists(),
+      createFormContent.exists(),
+      cancelButton.has({ disabled : false }),
+      saveButton.has({ disabled : true }),
+      createFormSection.find(Heading(createTitle)).exists(),
+      generalInformationButton.exists(),
+      nameInput.exists(),
+      generalInformationSection.find(HTML(nameRequiredAlert)).exists(),
+      cy.expect(overdueFineInput.has({ value : '0.00' }))
+    ]);
+  },
+
+  fillOverdueFine(value) {
+    cy.get('[name="overdueFine.quantity"]').invoke('val', '').clear()
+      .type(value, { force : true })
+      .blur();
+  },
+
+  fillMaxFine(value) {
+    cy.get('[name="maxOverdueFine"]').focus().invoke('val', '').clear()
+      .type(value, { force : true })
+      .blur();
+  },
+
+  fillOverdueRecall(value) {
+    cy.get('[name="overdueRecallFine.quantity"]').invoke('val', '').clear()
+      .type(value, { force : true })
+      .blur();
+  },
+
+  fillMaxRecall(value) {
+    cy.get('[name="maxOverdueRecallFine"]').invoke('val', '').clear()
+      .type(value, { force : true })
+      .blur();
+  },
+
+  save() {
+    cy.do(Button({ id: 'footer-save-entity' }).click());
+  },
+
+  checkAlert(alertText) {
+    cy.expect(editFineSection.find(HTML(alertText)).exists());
+  },
+
+  checkOverDueFineInCreating() {
+    cy.do([
+      intervalSelect.choose('day'),
+      overdueFineInput.focus(),
+      overdueFineInput.blur(),
+    ]);
+    this.checkAlert(intervalSetAlert);
+
+    cy.reload();
+    this.waitLoading();
+
+    cy.wait(1000);
+    this.fillMaxFine('1.00');
+    this.checkAlert(alertOverdueAmount);
+
+    this.fillOverdueFine('1.00');
+    this.save();
+    this.checkAlert(intervalAlert);
+
+    this.fillOverdueFine('-1.00');
+    this.checkAlert(overdueFineNegativeAlert);
+
+    this.fillOverdueFine('1.00');
+    this.fillMaxFine('0.00');
+    this.checkAlert(maximumOverDueFineAlert);
+  },
+
+  fillGeneralInformation(overdueFine, maxOverdueFine, recallFine, maxRecallFine) {
+    cy.wait(1000);
+    this.fillOverdueFine(overdueFine);
+    this.fillMaxFine(maxOverdueFine);
+    this.fillOverdueRecall(recallFine);
+    this.fillMaxRecall(maxRecallFine);
+
+    cy.do([
+      nameInput.fillIn(name),
+      intervalSelect.choose('day'),
+      Select({ name: 'overdueRecallFine.intervalId' }).choose('day'),
+    ]);
+  },
+
+  checkAfterSave(overdueFine, maxOverdueFine, recallFine, maxRecallFine) {
```

Comment:
rename pls to verifyCreatedFines or something like this
Do not use pls name like check after save, because this if not provide info about what function check